### DISCUSSION TITLE
Logging decorator

### DISCUSSION THREAD

**Discussion by robp94** [quarkusio/quarkus#39403] [Q&A] [answered]
I would like to enrich each log statement with certain things like a tenant ID. For this I must inject a request scoped bean. How could this be done? I mostly use the simplified logging like `Log.info`

**Comment by rquinio** [comment, marked as answer]
Easiest is probably to use the MDC API: https://quarkus.io/guides/logging#use-mdc-to-add-contextual-log-information

**Comment by robp94** [comment]
Thanks, @rquinio that's what I was looking for.
